pkgs/servers/http/envoy/default.nix

```
@@ -83,8 +83,8 @@ buildBazelPackage rec {
 
   fetchAttrs = {
     sha256 = {
-      x86_64-linux = "10f1lcn8pynqcj2hlz100zbpmawvn0f2hwpcw3m9v6v3fcs2l6pr";
-      aarch64-linux = "1na7gna9563mm1y7sy34fh64f1kxz151xn26zigbi9amwcpjbav6";
+      x86_64-linux = "sha256-l/BWUKmLr5m6OJDLMMuXA4w6eRX7jEzNMd8xV0vF+Rc=";
+      aarch64-linux = "sha256-/L/QC1nOLqLUGoXfNSXYqA+QHJ8euLLXPvfNEpTGBBc=";
```

```suggestion
      x86_64-linux = lib.fakeHash;
      aarch64-linux = lib.fakeHash;
```
let's debug via ofborg, I also now get different digest locally (maybe after rebases)